tests/workflows/ahm_analysis/test_ahm_analysis.py

```diff
@@ -17,7 +17,7 @@ def test_make_update_log_df(snake_oil_facade, snapshot):
     Note that this is now a snapshot test, so there is no guarantee that the
     snapshots are correct, they are just documenting the current behavior.
     """
-    with open_storage(snake_oil_facade.enspath, "w") as storage:
+    with open_storage("storage/snake_oil/ensemble", "w") as storage:
```

Comment:
Suggest rewriting the fixture to not use LibresFacade at all and give ErtConfig, as all the other use cases in this test is just using the config. We will still depend on ErtConfig here, but at least we drop the dependence on LibresFacade.